routes/web.php

```diff
@@ -218,6 +219,11 @@
 Route::get('/interactive-features/{slug}', [InteractiveFeatureExperiencesController::class, 'show'])->name('interactiveFeatures.show');
 Route::get('/interactive-features/kiosk/{slug}', [InteractiveFeatureExperiencesController::class, 'show'])->name('interactiveFeatures.show-kiosk');
 
+// Tour routes
+Route::get('/tours/builder', [CustomTourController::class, 'build']);
+Route::get('/tours/{id}', [CustomTourController::class, 'show']);
+Route::post('/api/tour', [CustomTourController::class, 'store']);
```

Comment:
Can you share the thinking behind not keeping this path in the `/tours` namespace?

Reply:
Now under /custom-tours/